scm-plugins/scm-git-plugin/src/main/java/sonia/scm/web/lfs/LfsAccessTokenFactory.java

```diff
@@ -80,15 +84,22 @@ AccessToken createWriteAccessToken(Repository repository) {
     PermissionCheck push = RepositoryPermissions.push(repository);
     push.check();
 
-    return createToken(Scope.valueOf(read.asShiroString(), pull.asShiroString(), push.asShiroString()));
+    int lfsAuthorizationTimeoutInMinutes = readLfsAuthorizationTimeoutInMinutes();
```

Comment:
```suggestion
    int lfsAuthorizationTimeoutInMinutes = getLfsAuthorizationTimeoutInMinutes();
```
I got confused for a second why you set the read timeout for the write access token. But you may leave it that way if you want.